src/app/api/services/content-types-info/content-types-info.service.ts

```diff
@@ -0,0 +1,71 @@
+import { Injectable } from '@angular/core';
+
+@Injectable()
+export class ContentTypesInfoService {
```

Comment:
not pretty sure what are you doing but shouldn't be the labels i18n instead of hardcoded english labels